docker-environment-driver/src/main/resources/isolatedDockerConfiguration/init.groovy

```diff
@@ -0,0 +1,9 @@
+import hudson.model.*;
+import jenkins.model.*;
+
+
+Thread.start {
+      sleep 10000
+      println "--> setting agent port for jnlp"
+      Jenkins.instance.setSlaveAgentPort(50000)
```

Comment:
this and 2 more scripts is just copy for @jbartece's jenkins which I don't want to change now as I didn't know what else is dependent on its exact setup and installation.